packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -109,9 +115,17 @@ function GridItem< Item >( {
 			<titleField.render item={ item } field={ titleField } />
 		) : null;
 
+	const isPicker = Boolean( picker );
+	const getIsItemClickable = useCallback(
+		() => ! isPicker && isItemClickable( item ),
+		[ isPicker, item, isItemClickable ]
+	);
+	const isClickable = getIsItemClickable();
```

Comment:
Is the callback necessary? Why not just:

```suggestion
	const isClickable = ! isPicker && isItemClickable( item );
```

Or is there a change in the expectation that `isItemClickable` is a cheap call, and thus something to now memoise?